### PR TITLE
Removes the 1s option from flashbangs

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

You can no longer set the fuse timer on flashbangs to one second

## Why It's Good For The Game

This is with 2(?) balance team members pre approval. (im unsure what the second member meant, but they seem fine with it?)

Flashbangs have started to become a quite cheesy method of closing the distance with people who lack one of the two forms of bang(ear) protection, and theres been a rather sharp increase of players treating others like a blob, and throwing upwards of 6 flashbangs on 1s timers, leading to no reaction time on the victim, removing the one second option should hopefully add a little skill requirement to 'insta flashing' as accurately timing 5/3 seconds with adrenaline in your system isn't the easiest thing todo.

not to mention if you lack eye protection the flashbang spam will likely blind you very quickly

this does not effect base grenades, only flashbangs
the options it gets now are as follows
0.1seconds ('truly' instant, if a clown gets ahold of your flashbox, you're in for a surprise. and as the bang part of the flashbang also effects you even if you have ear pro i think its fine.)
3 seconds
5 seconds

Hopefully this discourages/adds more skill to using flashbangs, as you will need to actually count in your head if you wish to surprise people with a flashbang

~~also technically a blob buff~~
## Testing

compiled, checked the flashbang options with a screwdriver
## Changelog
:cl:
tweak: you can no longer set flashbangs to a one second primer
/:cl:

### CODE REVIEW

**Review by DGamerL — CHANGES_REQUESTED** [review]
1. Please use the time defines
2. You didnt actually remove the 1 decisecond option

**Comment by 1080pCat** [comment]
> * 
> 
> * You didnt actually remove the 1 decisecond option

yea. only the 1 second option

**Review by Spaghetti-bit — APPROVED** [review]
While I'm not a fan of the switch statement and this could probably be condensed... it's functional. Approved.

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 1]
```diff
@@ -27,6 +27,20 @@
 		bang(T, src, range)
 	qdel(src)
 
+/obj/item/grenade/flashbang/screwdriver_act(mob/living/user, obj/item/I)
+	switch(det_time)
+		if(0.01 SECONDS)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/grenades/flashbang.dm:32]: 1 == 0.1 SECONDS, not 0.01
```suggestion
		if(0.1 SECONDS)
```

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 2]
```diff
@@ -27,6 +27,20 @@
...
+			det_time = 3 SECONDS
+			to_chat(user, "<span class='notice'>You set [src] for 3 second detonation time.</span>")
+		if(3 SECONDS)
+			det_time = 5 SECONDS
+			to_chat(user, "<span class='notice'>You set [src] for 5 second detonation time.</span>")
+		if(5 SECONDS)
+			det_time = 0.01 SECONDS
```
Comment (Contrabang) [inline code/game/objects/items/weapons/grenades/flashbang.dm:39]:
```suggestion
			det_time = 0.1 SECONDS
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/grenades/flashbang.dm (hunk 1 above)
Comment (1080pCat) [reply code/game/objects/items/weapons/grenades/flashbang.dm]: it didn't feel instant enough, but yeah probably a good idea to leave it for now

**Review by DGamerL — APPROVED** [review]
Why does nobody want to use `DECISECONDS` defines :(

**Comment by 1080pCat** [comment]
> Why does nobody want to use `DECISECONDS` defines :(

i thought the `SECONDS` were those 😭 would `DECISECONDS` be used for the `0.1` option?

**Comment by DGamerL** [comment]
> > Why does nobody want to use `DECISECONDS` defines :(
> 
> i thought the `SECONDS` were those 😭 would `DECISECONDS` be used for the `0.1` option?

0.1 seconds is 1 decisecond yeah!

**Review by Sirryan2002 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]
Should be fine

**Review by DGamerL — APPROVED** [review]

**Pull request merged**